### PR TITLE
User profile branch

### PR DESCRIPTION
**Pull request by avakrishn**

### CODE REVIEW

File: client/src/components/CryptosRanking/CryptosRanking.css [hunk 1]
```diff
@@ -18,6 +18,7 @@ thead {
   color: rgb(22, 190, 182);
 }
 
+/* Simon please add a class to your img tags so that it won't affect other pages */
```
Comment (simonnguyen3054) [inline client/src/components/CryptosRanking/CryptosRanking.css]: thanks for pointing this out @avakrishn

File: db/schema.sql [hunk 2]
```diff
@@ -136,7 +136,7 @@ CREATE TABLE users_purchases(
 	status_url VARCHAR(255) NULL,
 	qrcode_url VARCHAR(255) NOT NULL,
 	payment_received BOOLEAN NOT NULL DEFAULT FALSE,
-	permission VARCHAR(255) NOT NULL,
+	permission VARCHAR(255) NOT NULL DEFAULT "community",
```
Comment (simonnguyen3054) [inline db/schema.sql]: I like this that we defaults to community

File: routes/matched_friends.js [hunk 3]
```diff
@@ -35,4 +35,18 @@ var connection = mysql.createConnection({
     database: 'crypto_db'   
 });
 
+var id = 1;
+
+
+router.get("/matchedFriends/feed", function(req, res) {
+    connection.query(
+      "SELECT users_purchases.date_purchased, users_purchases.amount, deals.deal_name, users.username, users_profiles.photo, venues.venue_name, crypto_metadata.crypto_symbol AS crypto_symbol FROM users_purchases LEFT JOIN deals ON users_purchases.deal_id = deals.id LEFT JOIN users ON users_purchases.user_id = users.id LEFT JOIN crypto_info ON users_purchases.crypto_id = crypto_info.id LEFT JOIN crypto_metadata ON crypto_metadata_name = crypto_metadata.crypto_name LEFT JOIN venues ON venue_id = venues.id LEFT JOIN users_profiles ON users_profiles.user_id = users.id WHERE payment_received = ? AND NOT permission = ? AND users.id IN (SELECT matched_friend_id AS id FROM users_matched_friends WHERE user_id = ? AND both_accepted = 1) ORDER BY users_purchases.date_purchased DESC",
```
Comment (simonnguyen3054) [inline routes/matched_friends.js]: "AND NOT permission = ? AND users.id IN (SELECT matched_friend_id AS id FROM users_matched_friends WHERE user_id = ? AND both_accepted = 1)"

Can you explain this part @avakrishn?

File: client/src/components/Feed/Transactions/Transactions.js [hunk 4]
```diff
@@ -27,7 +27,9 @@ class Transactions extends Component {
     return (
       <div className="transactions-content">
         <CryptoRankings />
-        <FeedCard transactions={this.state.transactions}/>
+        <div className="w-50">
```
[the remark this answers is not in the document]
Comment (avakrishn) [reply client/src/components/Feed/Transactions/Transactions.js]: `w-50` is a Bootstrap class that makes the width of the div take up 50% of the parent div width

File: db/schema.sql (hunk 2 above)
Comment (avakrishn) [reply db/schema.sql]: I agree! I thought that we should have community be default and then users could specify further privacy if they prefer that. This way when we are inserting into the users_purchases table we don't have to worry about if the user did not select a permission value.

File: routes/matched_friends.js (hunk 3 above)
Comment (avakrishn) [reply routes/matched_friends.js]: @simonnguyen3054 This is a nested SQL query that allows me to select all the rows from the table where the users id's are all the matched friends.

**Pull request merged**